Frida/Script.swift

```diff
@@ -204,6 +204,43 @@ public class Script: NSObject, NSCopying {
             }
         }
 
+        if let dict = message as? [String: Any],
+            let type = dict["type"] as? String,
+            type == "send",
+            let payload = dict["payload"] as? [Any],
+            let kind = payload.first as? String,
+            kind == "frida:rpc",
+            let requestId = payload[1] as? Int,
+            let success = payload[2] as? String,
```

Comment:
We can count on `payload[1]` and `payload[2]` being of these types, given that `["frida:rpc", ...]` is considered reserved – though not currently validated by `send()`, but we should assume that the agent is well-behaved.